User Guide/Projects.md

```diff
@@ -858,20 +889,20 @@ To **Create topic**:
 
 ![Image not Available](/assets/Fig44.png)
 
-**Step 4:**	Select data classification. Click **Next**.
+**Step 4:** Select data classification. Click **Next**.
 
 ![Image not Available](/assets/Fig45.png)
 
-**Step 5:**	Select the project and enter topic name. Click **Next**.
+**Step 5:** Select the project and enter topic name. Click **Next**.
 
 ![Image not Available](/assets/Fig46.png)
 
-**Step 6:**	Fill in the following details of the topic.
+**Step 6:** Fill in the following details of the topic.
 
 - **Topic description** – Enter description to the topic
 - **Tags** – Add tags to the topic maximum of 5 tags
 - **Is approval needed for subscription requests?**
-  - **Yes** – Approval is needed from publisher before you subscribe 
+  - **Yes** – Approval is needed from publisher before you subscribe to MQTT Client
```

Comment:
"before you subscribe to MQTT Client" -> "before you can receive data to your MQTT Client"